### ISSUE TITLE
EditorConfig UI: Seemingly Duplicate entries

### ISSUE THREAD

**Issue by BhaaLseN** [dotnet/roslyn#53789]
**Version Used**: Visual Studio Community 2019 (Version 16.10.0)

**Steps to Reproduce**:

1. Open any project with an `.editorconfig` file (or create a new one)
2. Open the `.editorconfig` file with the new EditorConfig UI (vs. opening it with a text editor)
3. Look at the first few entries under "Space preferences"

**Expected Behavior**:
Entries **uniquely** identify what they do/affect, so users can make a decision on what they want there.

**Actual Behavior**:
The entry "Insert space between method name and its opening parenthesis" appears to be duplicated:
![image](https://user-images.githubusercontent.com/826052/120221763-4027a400-c23f-11eb-93ac-bbd3b421d34b.png)
But _actually_, the first one affects `csharp_space_between_method_declaration_name_and_open_parenthesis` ("method _declaration_ name") while the second one affects `csharp_space_between_method_call_name_and_opening_parenthesis` ("method _call_ name").

At first I also thought the same thing happening with the next two each, but on closer look I noticed that one said "_parameter_ list" while the other said "_argument_ list" (which could use some better distinction perhaps?)

**Comment by Joe4evr** [comment]
> (which could use some better distinction perhaps?)

🍝 How about if these options were separated into two sub-categories: `Call sites` and `Declaration sites`.

**Comment by BhaaLseN** [comment]
That would mean nested groups, right? Like a tree?

```
+- Space preferences
   +- Declaration sites
   |  |- Insert space between method name and its opening parentheses
   |  |- Insert space within parameter list parentheses
   |  \- Insert space within empty parameter list parentheses
   +- Call sites
      |- Insert space between method name and its opening parentheses
      |- Insert space within argument list parentheses
      \- Insert space within empty argument list parentheses
```

Not sure at the moment, it seems like _a_ solution but possibly not _the_ solution. Probably needs a bit more time to simmer (at least for me).
It would be better than the _current_ situation though.

Tools > Options handles it like this:
![image](https://user-images.githubusercontent.com/826052/120362062-f0aeaa00-c30a-11eb-8951-3500049b0c9a.png)
<sup>_Which, in hindsight, uses the exact same wording for the items and also shows "duplicates", if it weren't for the header before it_</sup>
So, basically similar to the proposal (albeit different wording), and it isn't _really_ a tree structure since the "header" is on first level. There is no "Space preferences" on top that groups _all_ the spacing related settings. However, there _is_ a distinct "Spacing" node to the right in the navigation tree.
If you'd consider that part as the whole hierarchy, it ends up similar to the above (except that its all in one view, not two).

In lack of better alternatives, I'd suggest using similar wording to the Options dialog, such as "Method declarations" and "Method calls" instead of "Declaration sites" and "Call sites".

**Comment by jmarolf** [comment]
duplicate of https://github.com/dotnet/roslyn/issues/60232
